xml/System.Web/HttpResponse.xml

```diff
@@ -1839,12 +1797,6 @@
 
  Any value can be a PICS label; ASP.NET does not validate the label. The maximum length of the string is 255 characters. For more information about PICS standards and syntax, see the [World Wide Web Consortium](https://www.w3.org/) web site.
 
-## Examples
- The following example is an ASP.NET page that displays an image. The page code calls the <xref:System.Web.HttpResponse.Pics%2A> method to set the HTTP `PICS-Label` header for the response. The string that is passed as the parameter to the <xref:System.Web.HttpResponse.Pics%2A> method represents a rating label generated from the Internet Content Rating Association (ICRA) Web site.
-
- [!code-aspx-csharp[System.Web.HttpResponse_Sample4#2](~/snippets/csharp/VS_Snippets_WebNet/System.Web.HttpResponse_Sample4/CS/page2cs.aspx#2)]
```

Comment:
Examples are removed here. Everything else in this file is whitespace changes or docs -> learn change.